efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -1,31 +1,35 @@
+# Lint as: python3
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Keras implementation of efficientdet."""
 import functools
 import tensorflow.compat.v1 as tf
-from tensorflow.python.keras.utils import conv_utils
```

Comment:
Two common issues here:
1. in general, we should try to avoid using any tensorflow or keras internal tools (specifically, using conv_utils would cause a warning internally).
2. our coding guideline recommends import modules only.  (from typing import ... is the only special case).

Reply:
:+1:

Reply:
Copy conv_utils to utils. Since keras needs to use it for global data_format

Reply:
Good idea. Since this PR is already very large, I am going to first merge this PR. Feel free to send a new PR to add this to utils.py. Thanks!